monkey/monkey_island/cc/server_utils/file_utils.py

```diff
@@ -0,0 +1,60 @@
+import os
+
+from monkey_island.cc.environment.utils import is_windows_os
+
+
+def expand_path(path: str) -> str:
+    return os.path.expandvars(os.path.expanduser(path))
+
+
+def has_expected_permissions(path: str, expected_permissions: int) -> bool:
+    if is_windows_os():
+        return _has_expected_windows_permissions(path, expected_permissions)
+
+    return _has_expected_linux_permissions(path, expected_permissions)
+
+
+def _has_expected_linux_permissions(path: str, expected_permissions: int) -> bool:
+    file_mode = os.stat(path).st_mode
+    file_permissions = file_mode & 0o777
+
+    return file_permissions == expected_permissions
+
+
+def _has_expected_windows_permissions(path: str, expected_permissions: int) -> bool:
+    # checks that admin has any permissions, user has `expected_permissions`,
+    # and everyone else has no permissions
```

Comment:
>checks that admin has any permissions

I think this comment is misleading. If I'm reading the code correctly, we don't check the admin permissions at all, we just skip/ignore them. We can probably nix this comment.